pkg/apis/psmdb/v1/perconaservermongodbrestore_types.go

```diff
@@ -82,7 +83,7 @@ func (r *PerconaServerMongoDBRestore) CheckFields() error {
 			return errors.New("backupSource destination is required")
 		}
 
-		if r.Spec.BackupSource.Destination[:5] != "s3://" {
+		if r.Spec.BackupSource.S3 != nil && !strings.HasPrefix(r.Spec.BackupSource.Destination, "s3://") {
```

Comment:
should we add validation for the azure too?

Reply:
@egegunes I don't think we should. Azure `BackupSource.Destination` doesn't have any prefixes as s3 has